Tools/scripts/pathfix.py

```diff
@@ -164,12 +179,27 @@ def fix(filename):
     # Return success
     return 0
 
+
+def parse_shebang(shebangline):
+    shebangline = shebangline.strip()
+    end = len(shebangline)
+    start = shebangline.find(b' -', 0, end)
```

Comment:
"end" argument is useless: shebangline.find(b' -') behaves the same and is simpler. Remove 0 and end.